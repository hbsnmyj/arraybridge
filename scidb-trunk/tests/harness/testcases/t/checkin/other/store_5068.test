# ticket:5068 revealed an overlook in store().
# Previously, store() only checked for naming conflicts among attributes and among dimensions.
# Some queries crashed the engine, when they generated conflicts between a dimension name and an attribute name.
# Both queries in this test checks that case.

--setup
--test

store(cross_join(build(<a:float>[b=0:0,1,0],1), apply(build(<a:float>[c=0:0,1,0],1), b, 1), b, c), a5068)
store(unpack(sort(build(<n:double>[i=1:1,1,0],1) as A), n), b5068)

--cleanup
remove(a5068)
remove(b5068)
