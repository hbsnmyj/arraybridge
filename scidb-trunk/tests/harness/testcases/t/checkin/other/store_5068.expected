{b_2} a,a_2,b
{0} 1,1,1

{n_3} n,n_2
{0} 0,1

Query was executed successfully

Query was executed successfully

